docs/operations.rst

```diff
@@ -14,42 +14,169 @@ Provisioning
 ------------
 
 It is also possible to register pre-existing PostgreSQL instances with a
-pg_auto_failover monitor. The ``pg_autoctl create`` command honors the ``PGDATA``
-environment variable, and checks whether PostgreSQL is already running. If
-Postgres is detected, the new node is registered in SINGLE mode, bypassing
-the monitor's role assignment policy.
+pg_auto_failover monitor. The ``pg_autoctl create`` command honors the
+``PGDATA`` environment variable, and checks whether PostgreSQL is already
+running. If Postgres is detected, the new node is registered in SINGLE mode,
+bypassing the monitor's role assignment policy.
 
-.. _pg_auto_failover_security:
+Upgrading pg_auto_failover
+--------------------------
 
-Security
```

Comment:
I guess this could be deleted, because security now has it's own complete page?

Reply:
Yes.